### DISCUSSION TITLE
I have a custom post_type, which is called news, I would like to bring these posts in the calls.

### DISCUSSION THREAD

**Discussion by bortoli99** [wp-graphql/wp-graphql#2730] [Q&A] [answered]
In the WP that I'm adding the plugin, post_type = 'post' is no longer used, only post_type = 'news' or post_type = 'article_columnist'. However, I just want to pull the news through the API

**Comment by justlevine** [comment, marked as answer]
Hey @bortoli99 👋

To hide your `post` post type, you can filter the post type args and set `show_in_graphql` to `false`. 

Same with `news`, and `article_columnist`, except instead of filtering, you'd add `show_in_graphql` to the `$args` in your `register_post_type()` call.

You can learn more [in this docs article](https://www.wpgraphql.com/docs/custom-post-types)

**Reply by bortoli99** [reply]
It worked.
Thanks for your help, love you man.
